webstruct/webannotator.py

```diff
@@ -264,6 +186,107 @@ def _set_base(tree, baseurl):
     head = _ensure_head(tree)
     head.insert(0, Element("base", href=baseurl))
 
+TagPosition = namedtuple('TagPosition', ['element',
```

Comment:
It seems this namedtuple is an implementation detail of to_webannotator function - what do you think about making it private, i.e. renaming to `_TagPosition`?